gluster/file/examples/claim.yaml

```diff
@@ -1,7 +1,7 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: claim1
+  name: glusterfs-claim
```

Comment:
If we change the name, its also better to rename the file name.

Reply:
Fair enough.